resources/master-kubelet.service

```diff
@@ -39,21 +39,16 @@ ExecStart=/usr/bin/docker \
     --volume /var/lib/calico:/var/lib/calico:rw \
     --entrypoint /usr/local/bin/kubelet \
   "${kubelet_image_url}:${kubelet_image_tag}" \
+    --allow-privileged \
+    --config=/etc/kubernetes/config/master-kubelet-conf.yaml \
     --kubeconfig=/var/lib/kubelet/kubeconfig \
     --node-labels=role=master,node-role.kubernetes.io/master="" \
     --register-with-taints=node-role.kubernetes.io/master=:NoSchedule \
-    ${feature_gates == "" ? "" : "--feature-gates=${feature_gates}"} \
     --container-runtime=docker \
     --network-plugin=cni \
     --cni-bin-dir=/opt/cni/bin \
     --cni-conf-dir=/etc/cni/net.d \
-    --allow-privileged \
-    --pod-manifest-path=/etc/kubernetes/manifests \
-    --minimum-container-ttl-duration=6m0s \
```

Comment:
I don't see this flag replicated in the yaml config

Reply:
`Flag --minimum-container-ttl-duration has been deprecated, Use --eviction-hard or --eviction-soft instead. Will be removed in a future version.`

No, it was only put in recently coping OpenShift config, so I just removed it.

Reply:
LGTM then!